engine/engine_test.go

```diff
@@ -238,7 +238,7 @@ func TestDeal(t *testing.T) {
 		}
 
 		if len(player.hand) != 13 {
-			t.Errorf("expected hand length to be 13, recieved %d", len(player.hand))
+			t.Errorf("expected hand length to be 13, received %d", len(player.hand))
```

Comment:
Test deal is now failing on my computer. Perhaps we can figure this out together?

Reply:
weird! It's passing for me. Lets go through it together, yes.